### ISSUE TITLE
Convolutional Neural Network(Detecting single set of images)

### ISSUE THREAD

**Issue by VISWESWARAN1998** [keras-team/keras#11027]
Consider that, I have images of set of watches and I am willing to identify further watches using CNN to identify it, but there is a problem in it. What the problem was is in order for the classification algorithm to work we need another class(binary classification).

So I have added another class, now I have two sets of images. The first set is the images of watches alone and the another set is some random images(anything without watch on it). I am getting the accuracy very low.

Can you give me an example of how to use CNN to achieve it? with only one set of images to identify further images? like object identification in CNN.
